### DISCUSSION TITLE
Upgrade Quasar from Vue 3.0.0 to vue 3.3.0

### DISCUSSION THREAD

**Discussion by Alfredoarnoldo** [quasarframework/quasar#16802] [General - Components / Directives / etc] [answered]
Hello, 

I would like to know if it is possible to upgrade quasar to the newest version of vue3 ? (eg 3.3 or 3.4)
Right now, even after an upgrade to the last quasar version, I can always see "vue": "^3.0.0" in my package.json.
If it is possible, how can we do that to take advantage of the latests vue3 fonctionnality in Quasar such as Definemodel?
Is just upgrading vue3 in the quasar project is enough?

thank you very much!

**Comment by rstoenescu** [comment, marked as answer]
Hi,

The short answer: yes, you can upgrade to any version of Vue 3. You do that with your package manager of choice (yarn/npm/etc).
The caret character (`^`) in `^3.0.0` means latest v3, it does not necessarily mean "exactly 3.0.0".

It would be a good idea to master your package manager CLI commands. Otherwise, just delete the lock file generated by your package manager (which can be yarn.lock or package-lock.json etc etc) AND your node_modules and install again. By having that caret character you'll get latest Vue 3.

**Reply by Alfredoarnoldo** [reply]
Thank you very much. I just installed the latest version of vue 3 (3.3.4) and my quasar project still works perfectly.
